common/changes/@microsoft/rush/sachinjoseph-SupportForPnpm4_2019-10-09-02-14.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Support PNPM 4 on Rush",
```

Comment:
"Add support for PNPM 4.x"

How confident are we that it works reliably?  If this is not very well tested, maybe say "Add support for PNPM 4.x (Experimental)"

Reply:
Yes, let's add the experimental tag for now. It's possible I am missing something.